sdk/src/invariant.ts

```diff
@@ -189,6 +192,23 @@ export class Invariant {
     )
   }
 
+  async getTickmap(key: PoolKey): Promise<Tickmap> {
+    return {
+      bitmap: new Map(
```

Comment:
Note not related to the task objective but a matter to consider for future tasks. There is often a pattern of fetching and then mapping the result. The results can be mapped using utility functions. This will allow keeping the mapping consistent, as there will likely be multiple entrypoints fetching the same data but with different limits, etc.